free-science-books-de.md

```diff
@@ -0,0 +1,14 @@
+# Fachrichtungen
+
+## Elektronik
+* [Das Elektronik-Labor - Labortagebuch](http://www.elektronik-labor.de/Labortagebuch/Labortagebuch.html)
+
+##Informatik
+* [Informatik-Schule](http://www.inf-schule.de/)
+* [Schule Regensburg - Informatik](http://www.schulen.regensburg.de/wvsgym/images/Faecher/Informatik/)
+
+##Mathe
+* [Mathe Prisma](http://www.matheprisma.uni-wuppertal.de/index.htm)
+
+##Musik
+* [Klavier spielen - Grundlagen, Übungen, Praxistipps](http://foppde.uteedgar-lins.de/foppde.html#copy)
```

Comment:
Again. Dpending on the point of view you can consider Music, Electronics and Mathematics a science or not.